src/test/java/org/cactoos/text/TextOfTest.java

```diff
@@ -311,4 +316,52 @@ public void printsStackTraceFromArray() {
             )
         );
     }
+
+    @Test
+    public void readsLocalDateFormattedWithFormatString() throws IOException {
+        final LocalDate date = LocalDate.of(2017, 12, 13);
+        MatcherAssert.assertThat(
```

Comment:
@Iprogrammerr use cactoos-matchers `Assertion` instead of `assertThat`

Reply:
@victornoel Correct me, if I am wrong, but I do not see any cactoos-matchers release were this i object is available. The latest version is 0.12 and there is no Assertion.

Reply:
@Iprogrammerr 0.13 has been released now :)

Reply:
@victornoel Yes, let's wait for #1023.